src/energyplus/ForwardTranslator/ForwardTranslateThermalZone.cpp

```diff
@@ -856,10 +843,15 @@ namespace energyplus {
       }
 
       boost::optional<IdfObject> dsoaList;
+      bool needToRegisterDSOAList = false;
+      bool atLeastOneDSOAWasWritten = true;
```

Comment:
Two things:
* A DSOA:SpaceList with zero DSOAs made E+ crash. So I protect against it
* We need to write an ALL ZERO DSOA in case none is explicitly specified or E+ uses 0.0094 m3/s-person if blank!